devicetypes/sky-nie/min-smart-plug-dimmer.src/min-smart-plug-dimmer.groovy

```diff
@@ -382,14 +508,14 @@ private logTrace(msg) {
 
 def on() {
     logDebug "on()..."
-
-    return [ basicSetCmd(0xFF) ]
+    sendHubCommand(basicSetCmd(0xFF))
```

Comment:
You don't have to call sendHubCommand here. It should be enough to simply `return basicSetCmd(0xFF)`
This remark applies to: `configure(), ping(), on(), off(), refresh(), setLevel() `

Reply:
OK! I modified it.